Dockerfile.pipelines

```diff
@@ -132,6 +132,9 @@ RUN if [ $installWithPip == "true" ] ; then \
       fi ; \
       if [ $RUN_EXAMPLES == "true" ] ; then \
         bash scripts/create_ipyparallel_mpi_profile.sh; \
+        git clone https://github.com/opesci/opescibench.git /tmp/opescibench ; \
```

Comment:
I know it an extra block but I'd like this in a separate `if` to separate from MPI installs